README.md

```diff
@@ -33,7 +33,7 @@ FLIGHTCTL_SERVER=<api_server_url> npm run dev
 If the backend, or OIDC provider is running self-signed certs, you will need to disable the verification via env variables:
 
  - `FLIGHTCTL_SERVER_INSECURE_SKIP_VERIFY='true'` - to disable verification of backend certs
- - `OIDC_INSECURE_SKIP_VERIFY='true'` - to disable verification of OIDC certs
+ - `AUTH_INSECURE_SKIP_VERIFY='true'` - to disable verification of auth server certs
```

Comment:
The default for `AUTH_INSECURE_SKIP_VERIFY` was change to true, so the instructions would need to be reversed (set the value to false to enable verification),  or keep the default to `false`.

Reply:
we still want to default to false, fixed